com.github.naaando.lyrics.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id": "com.github.naaando.lyrics",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.naaando.lyrics",
+    "build-options": {
+        "env": {
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+        }
+    },
+    "finish-args": [
+        "--socket=x11",
```

Comment:
You need to add --share=ipc if you're using x11 per the docs http://docs.flatpak.org/en/latest/sandbox-permissions.html

Reply:
Confusingly it's working without it, I should be using this for display a GUI or this is meant for window managers, compositors, etc?

Reply:
It works, it just doesn't use X11 shared memory acceleration. You want the ipc option.